### DISCUSSION TITLE
Delete Test store

### DISCUSSION THREAD

**Discussion by leonkosterss** [swellstores/community#30] [Help] [answered]
Hello Swell,

Can someone help me with deleting a test store? I wanted to test creating Orders with the API. But I forgot, it has a 2% fee. I think best is to delete the store. How can I do this?

Thanks in Advance!

**Comment by papertokyo** [comment, marked as answer]
@leonkosterss We only charge a percentage fee on orders using a live payment gateway, or where actual money has been paid. If you're just testing, that's not considered billable.

If you're unsure about your particular situation you can open a chat from your dashboard so the support team has all your account details and can give you a more definitive answer based on your order activity—or delete the store if you prefer.

**Reply by ericingram** [reply]
To add some clarity, we also charge a % fee when manual payment gateways are used, because there's no way for the platform to know whether it's live or not in that case. However, when using a test credit card or alternative payment method in test mode, for example, it will not incur a fee at all.

That said, we're going to introduce a dedicated Test environment for each store, soon, that will solve this problem once and for all.

**Reply by steveninety** [reply]
I am being charged for test sales through my own Stripe integration in test mode. My store is far from ready and not publicly accessible. Is there any way I can contact support about this? In the dashboard it tells me to upgrade before I can contact support...

**Comment by leonkosterss** [comment]
Thanks for your answers! :) 

To be sure: Orders that are made by the API, are not charged, because it does not use a payment gateway, right?

The confusion started, because of a mail what suggests upgrading. Maybe don't sell this mail, if the amount of those orders are made without a payment gateway.

@papertokyo Can you tell me how I can delete a store? I couldn't find it.

@ericingram Great to hear you're working on test environments! Thats a functionality many headless systems (including ecommerce) are missing.

**Reply by ericingram** [reply]
Any API that's charged using either a) A "live" payment gateway or b) A manual payment method - either by API or manually - are considered as "live sales". But totally admit it's confusing on the manual payment method point. Test environments will fix that!

Here's a sneak peek:

![image](https://user-images.githubusercontent.com/952134/207699375-7f1cab96-a3ff-4d3c-bb99-c22a12fac3ed.png)

**Reply by vettloffah** [reply]
Test environments will be a really nice feature.

**Comment by dannyleungkc** [comment]
I also had the same request and no one actually responded the query directly. There seems no way to delete a test store...

**Reply by ghost** [reply]
go to admin ,and on left side you have Storefront - Settings .Store password and put something strong for now

**Comment by adrrian17** [comment]
Hi, is there a way to delete a store?

**Reply by jramoyo-serenade** [reply]
@ericingram @papertokyo How can we delete a test store?

Edit: Ok, looks like there is no such thing as a "Test Store". Each store is independent of each other and can have either 1 test mode and 1 live mode.

The way stores are created can be misleading because it is attached to your account and it immediately says "Test Mode". It gives the impression that you can create an arbitrary number of "Test Stores", when in fact each store starts a separate 14 day trial.

Is there at least a way to delete "Expired Stores"? So that it does not show up in the account?

Also, do you have any recommendation for development teams that work on multiple non-production (non-live) environments? i.e. can we have multiple "Test Modes" so that our environments won't have overlapping data?

**Reply by ericingram** [reply]
@jramoyo-serenade this is good feedback. We don’t yet have a way for you to fully delete a store, but we can manually disassociate it from your user account so you don’t have to see it. cc @steven-k1

**Reply by steven-k1** [reply]
@jramoyo-serenade Can you let me know the IDs of the accounts that you would like to have disassociated with your user?

**Reply by jramoyo-serenade** [reply]
@steven-k1 thanks, would you be able to disassociate `test-local` from our account please?

link:
https://test-local.swell.store/

**Reply by steven-k1** [reply]
@jramoyo-serenade You should be all set now, but let me know if you still see the indicated account.

**Reply by jramoyo-serenade** [reply]
Thank you @steven-k1, it's no longer visible.
